infra/tf/better_uptime/main.tf

```diff
@@ -0,0 +1,53 @@
+provider "betteruptime" {
+	api_token = module.betteruptime_secrets.values["better_uptime/token"]
```

Comment:
This secret doesn't match the secret read in betteruptime_secrets. Pretty sure this was committed and untested, which i think is what i asked for :P